extensions/schema-compare/src/test/schemaCompare.test.ts

```diff
@@ -79,22 +84,22 @@ describe('SchemaCompareResult.start', function (): void {
 	it('Should start with the source as database', async function (): Promise<void> {
 		let sc = new SchemaCompareTestService();
 
-		let result = new SchemaCompareMainWindow(sc, mockExtensionContext.object);
-		await result.start({connectionProfile: mockConnectionProfile});
+		let result = new SchemaCompareMainWindow(testContext.apiWrapper.object, sc, mockExtensionContext.object);
+		await result.start({connectionProfile: mockIConnectionProfile});
 		let promise = new Promise(resolve => setTimeout(resolve, 5000)); // to ensure comparison result view is initialized
 		await promise;
 
 		should.notEqual(result.sourceEndpointInfo, undefined);
 		should.equal(result.sourceEndpointInfo.endpointType, mssql.SchemaCompareEndpointType.Database);
-		should.equal(result.sourceEndpointInfo.serverName, mockConnectionProfile.serverName);
-		should.equal(result.sourceEndpointInfo.databaseName, mockConnectionProfile.databaseName);
+		should.equal(result.sourceEndpointInfo.serverName, mockIConnectionProfile.serverName);
```

Comment:
ah goo that we already have a test to create SCMainWindow with a Source Endpoint - can we add a test to update/change the endpoint too?

Also in this test there should be a check that target endpoint is empty/undefiled - right?

Reply:
I will add the suggested test in another round of test PR.
As for second, as discussed, the check already exists.